src/QuickOrder.tsx

```diff
@@ -0,0 +1,134 @@
+
+import React, { useState } from 'react';
+import { useTranslation, useCartData } from './app-state';
+import { getProductById, bulkAdd } from './service';
+import { ReactComponent as ClearIcon } from './images/icons/ic_clear.svg';
+
+import './QuickOrder.scss'
+
+export const QuickOrder: React.FC = (props) => {
+  const { t } = useTranslation();
+  const { updateCartItems } = useCartData();
+
+  const defaultItem = {
+    code: '', quantity: 0, isInvalid: false,
+  };
+
+  const defaultItemsCount = 10;
+  const additionalItemsCount = 2;
+
+  const [items, setItems] = useState(Array(defaultItemsCount).fill(defaultItem).map((item, index) => ({ ...item, key: `quick-order-sku-${index}` })));
+  const [error, setError] = useState('');
+  const [showLoader, setShowLoader] = useState(false);
+
+  const handleUpdate = (index:number, arg:{[key: string]: any}[]) => {
+    const itemsArr:any[] = [...items];
+    itemsArr[index] = {...itemsArr[index]};
+    arg.map(el => {
+      return itemsArr[index][Object.keys(el)[0]] = Object.values(el)[0];
+    });
+    setItems(itemsArr);
+  };
+
+  const handleBlur = (index:number, code:string) => {
+    if (code === '') return;
+    getProductById(code)
```

Comment:
We should be able to pass product ID's directly to the form without the need for this check. Is this check purely done to validate if the product exists? If so, let's discuss with Yury and Bonnie on a better experience.

We'll have to update this anyway to accept skus vs product IDs.